src/playlist.js

```diff
@@ -256,7 +256,7 @@ export const safeLiveIndex = function(playlist) {
  * @returns {Number} the end time of playlist
  * @function playlistEnd
  */
-export const playlistEnd = function(playlist, expired, useSafeLiveEnd) {
+export const playlistEnd = function(playlist, expired, useSafeLiveEnd, suggestedPresentationDelay) {
```

Comment:
The jsdoc for `useSafeLiveEnd` may need to be changed, since it assumes a value for presentation delay.